### PR TITLE
i1044: Update repository to record set type

### PR DESCRIPTION
**Pull request by MartinEden**
First part of https://vimc.myjetbrains.com/youtrack/issue/VIMC-1044

### CODE REVIEW

File: scripts/start-database.sh [hunk 1]
```diff
@@ -17,6 +16,13 @@ echo " - $migrations_image"
 docker pull $db_image
 docker pull $migrations_image
 
+# If the database is already running, stop it
+if docker top $NAME &>/dev/null; then
+    echo "Stopping database"
+    docker stop $NAME
+    sleep 1s
+fi
```
Comment (MartinEden) [inline scripts/start-database.sh]: This isn't specific to this PR; it's just a handy thing I've been meaning to add for a while.

File: src/app/src/main/kotlin/org/vaccineimpact/api/app/errors/UnexpectedError.kt [hunk 2]
```diff
@@ -12,7 +12,7 @@ val supportAddress = Config["contact.support"]
 // The purpose of the code, even though it's not used anywhere else, is that the full text of the error gets logged.
 // So if a user gives you a code like UAA-BBB-CCC you can then search through the logs to find and see the associated
 // stack trace.
-class UnexpectedError private constructor(exception: Exception) : MontaguError(500, listOf(ErrorInfo(
+class UnexpectedError private constructor() : MontaguError(500, listOf(ErrorInfo(
```
Comment (MartinEden) [inline src/app/src/main/kotlin/org/vaccineimpact/api/app/errors/UnexpectedError.kt]: Just removing an unused parameter; again not strictly about this PR.

File: scripts/start-database.sh (hunk 1 above)
Comment (littledice) [reply scripts/start-database.sh]: will this not error if the db container isn't running?

Comment (littledice) [reply scripts/start-database.sh]: lol sorry github just showed me lines 21-23, I see that you are checking for that!

**Pull request merged**